styles/base.css

```diff
@@ -58,5 +58,10 @@
         @apply my-3
     }
 
+    .custom-grid {
```

Comment:
Hvorfor ikke bruke tailwind der du bruker denne klassen?

Reply:
Jeg opplevde denne som en enkel og dynamisk måte å få kortene til å fordele seg over riktig antall rader etter skjermstørrelse uten å måtte definere ulikt for s: m: osv. Tar gjerne et forlag om du har en annen løsning du syns er god. Denne delen er for de kortene med tid og rom osv, så de er ikke synlige i denne versjonen.